server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -683,10 +718,33 @@ def _get_single_condition_criterion(
                     return column_field.isnotnull()
 
             case DateBoundCondition():
-                if condition.operator == "until":
-                    return column_field <= condition.value
-                elif condition.operator == "from":
-                    return column_field >= condition.value
+
+                if isinstance(condition.value, (RelativeDate, datetime, str)):
+                    if isinstance(condition.value, RelativeDate):
+                        value_str_time = (
+                            evaluate_relative_date(condition.value)
+                            .astimezone()
+                            .strftime("%Y-%m-%d %H:%M:%S")
+                        )
+                    elif isinstance(condition.value, datetime):
+                        value_str_time = condition.value.astimezone().strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
This converts the TZ to the system time, and forces the TZ on naive objects. I thought we agreed on UTC ?